db/migration/V020__insert_default_activities.sql

```diff
@@ -0,0 +1,128 @@
+  /* week5 -5 */ 
+  ('Self relection', '', 5, 5, '10:00:00', '11:00:00', 60, 4, 1),
+  ('1:1 mentorship', '', 5, 5, '11:10:00', '13:10:00', 120, 4, 1),
+  /* week6 - 1*/ 
+  ('Morning standup', '', 6, 1, '10:00:00', '11:00:00', 60, 3, 1),
+  ('Sprint Planing', 'Planing the schedule for the week and assigning tickets', 6, 1, '11:10:00', '12:00:00', 50, 1, 1), 
+  /* week6 - 2 */ 
+  ('Self assignment', '', 6, 2, NULL, NULL, NULL, 2, 1),
+  ('Working on the ticket', 'Working on the assigned ticket', 6, 2, '10:00:00','24:00:00', 840, 1, 1),
+  /* week6 - 3 */ 
+  ('Morning standup', 'Check the progress', 6, 3, '11:00:00', '12:00:00', 60, 3, 1),
+  ('Working on the ticket', 'Working on the assigned ticket', 6, 3, '12:10:00','24:00:00', 710, 1, 1),
+  /* week6 - 4 */ 
+  ('Game time', '', 6, 4, '10:00:00', '10:30:00', 30, 1, 1),
+  ('Working on the ticket', 'Working on the assigned ticket', 6, 2, '11:00:00','24:00:00', 780, 1, 1),
+  /* week6 - 5 */ 
+  ('Self relection', '', 6, 5, '10:00:00', '11:00:00', 60, 4, 1),
+  ('1:1 mentorship', '', 6, 5, '11:10:00', '13:10:00', 120, 4, 1),
+  /* week7 - 1 */ 
+  ('Morning standup', '', 7, 1, '10:00:00', '11:00:00', 60, 3, 1),
+  ('Sprint Planing', 'Planing the schedule for the week and assigning tickets', 7, 1, '11:10:00', '12:00:00', 50, 1, 1), 
+  /* week7 - 2 */ 
+  ('Self assignment', '', 7, 2, NULL, NULL, NULL, 2, 1),
+  ('Working on the ticket', 'Working on the assigned ticket', 7, 2, '11:00:00','24:00:00', 780, 1, 1),
+  /* week7 - 3 */ 
+  ('Morning standup', 'Check the progress', 7, 3, '11:00:00', '12:00:00', 60, 3, 1),
+  ('Working on the ticket', 'Working on the assigned ticket', 7, 3, '12:10:00','24:00:00', 710, 1, 1),
+  /* week7 - 4 */ 
+  ('Game time', '', 7, 4, '10:00:00', '10:30:00', 30, 1, 1),
+  ('Working on the ticket', 'Working on the assigned ticket', 7, 4, '11:00:00','24:00:00', 780, 1, 1),
+  /* week7 - 5 */ 
+  ('Self relection', '', 7, 5, '10:00:00', '11:00:00', 60, 4, 1),
+  ('1:1 mentorship', '', 7, 5, '11:10:00', '13:10:00', 120, 4, 1),
+  /* week8 - 1 */ 
+  ('Morning standup', '', 8, 1, '10:00:00', '11:00:00', 60, 3, 1),
+  ('Sprint Planing', 'Planing the schedule for the week and assigning tickets', 8, 1, '11:10:00', '12:00:00', 50, 1, 1), 
+  /* week8 - 2 */ 
+  ('Self assignment', '', 8, 2, NULL, NULL, NULL, 2, 1),
+  ('Working on the ticket', 'Working on the assigned ticket', 8, 2, '11:00:00','24:00:00', 780, 1, 1),
+  /* week8 - 3 */ 
+  ('Morning standup', 'Check the progress', 8, 3, '11:00:00', '12:00:00', 60, 3, 1),
+  ('Working on the ticket', 'Working on the assigned ticket', 8, 3, '12:10:00','24:00:00', 710, 1, 1),
+  /* week8 - 4 */   
+  ('Game time', '', 8, 4, '10:00:00', '10:30:00', 30, 1, 1),
+  ('Working on the ticket', 'Working on the assigned ticket', 8, 4, '11:00:00','24:00:00', 780, 1, 1),
+  /* week8 - 5 */   
+  ('Self relection', '', 8, 5, '10:00:00', '11:00:00', 60, 4, 1),
+  ('1:1 mentorship', '', 8, 5, '11:10:00', '13:10:00', 120, 4, 1);
+
```

Comment:
Please shorten up the space at the end of the file to be a single blank line.

Reply:
Fix it in commit https://github.com/OpenTree-Education/rhizone-lms/pull/421/commits/878d93266fffdbc0d4f42762e95680441fcca4d3